notebooks/2_feature_engineering.ipynb

```diff
@@ -74,27 +65,34 @@
     "* Selective provides an easy-to-use API for supervised and unsupervised feature selection methods.\n",
     "* In unsupervised fashion, given a set of users, important features can be identified according to variance, correlation and statistical measures. \n",
     "* In supervised fashion, given a set of users _and_ the interaction label (e.g., click on _any item_), important features can be identified according to a linear or non-linear model. \n",
-    "* Let's explore a quick start example. "
-   ]
+    "* Let's install the library and explore a quick start example."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Install Selective\n",
```

Comment:
FYI, previously there was one Code Cell at the top that installed all 3 libraries at the same time. 

Based on feedback from USer Group, I noticed that this creates a blocker sometimes. Anytime one library fails to install, the user is stuck at the top of the notebook. 

Instead, I distributed the install cells on top of each section. You wanna use Selective, then all you need to worry about is selective install (and you won''t be blocked if, say, seq2pat fails install due to C++ backend..)